docs/create_executable.md

```diff
@@ -0,0 +1,23 @@
+# Create executable
+
+Create an executable of dot for different OS.
+
+## Windows
+
+Run these commands
+
+```
+cd path/to/dot
+conda activate dot
+pyinstaller --noconfirm --onedir --name "dot" --add-data "src/dot/fomm/config;dot/fomm/config" --add-data "src/dot/simswap/models;dot/simswap/models" --add-data "path/to/Anaconda3/envs/dot/Lib/site-packages;." --add-data "configs;configs/" --add-data "data;data/" --add-data "saved_models;saved_models/" src/dot/ui/ui.py
```

Comment:
you can use this command to fetch the site packages folder: 

```
python -c "import site; print(''.join(site.getsitepackages()))"
```

users might not know how to find the path. I suggest you create a bash script as well to create executable.

Reply:
ah! no need for a bash script then. is it possible to save the output of the above python command in a variable and then use it for the `pyinstaller` command?

Reply:
I think we are over complicating this for windows. I will add a comment about finding the site packages folder

Reply:
that works too.